app/models/mixins/archived_mixin.rb

```diff
@@ -2,11 +2,14 @@ module ArchivedMixin
   extend ActiveSupport::Concern
 
   included do
+    scope :deleted, -> { where(:deleted => true) }
+    scope :not_deleted, -> { where(:deleted => false) }
```

Comment:
I think it should not matter, since the migration and the scope change will be 2 PRs, there will always be a moment where it's broken before everything is merged :-)

But we can do that if you want. :-)